src/Compilers/VisualBasic/vbc/app.manifest

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I think we can share a single manifest file.

I *think* the name of the manifest file needs to match the name of the exe: `csc.exe.manifest`, `vbc.exe.manifest`, etc.

We need a build task that updates `assemblyIdentity`. This task can take a shared manifest file as an input and produce manifest file that's specific to the project and with `assemlyIdentity` set properly:

```xml
<Target Name="_CalculateSubstituteVariablesInApplicationManifestInputsOutputs">
    <PropertyGroup>
      <_OriginalApplicationManifestPath>$([System.IO.Path]::Combine('$(MSBuildProjectDirectory)', '$(ApplicationManifest)'))</_OriginalApplicationManifestPath>
      <_UpdatedApplicationManifestPath>$(IntermediateOutputPath)$(AssemblyName).exe.manifest</_UpdatedApplicationManifestPath>
    </PropertyGroup>
  </Target>

  <!--
    Sets the values of "version" and "name" attributes in assemblyIdentity element in the application manifest file
    with values $(AssemblyVersion) and $(AssemblyName) to avoid hardcoding those in the manifest file.
  -->
  <Target Name="_SubstituteVariablesInApplicationManifest"
          BeforeTargets="_SetExternalWin32ManifestProperties;_SetEmbeddedWin32ManifestProperties"
          DependsOnTargets="_CalculateSubstituteVariablesInApplicationManifestInputsOutputs;GetAssemblyVersion"
          Inputs="$(_OriginalApplicationManifestPath)"
          Outputs="$(_UpdatedApplicationManifestPath)"
          Condition="'$(ApplicationManifest)' != ''">

    <Copy SourceFiles="$(_OriginalApplicationManifestPath)" DestinationFiles="$(_UpdatedApplicationManifestPath)" />

    <PropertyGroup>
      <_Namespace>
        <Namespace Prefix="asm" Uri="urn:schemas-microsoft-com:asm.v1" />
      </_Namespace>
    </PropertyGroup>

    <!-- Update version attribute -->
    <XmlPoke XmlInputPath="$(_UpdatedApplicationManifestPath)"
             Value="$(AssemblyVersion)"
             Query="/asm:assembly/asm:assemblyIdentity/@version"
             Namespaces="$(_Namespace)"/>

    <!-- Update name attribute -->
    <XmlPoke XmlInputPath="$(_UpdatedApplicationManifestPath)"
             Value="$(AssemblyName)"
             Query="/asm:assembly/asm:assemblyIdentity/@name"
             Namespaces="$(_Namespace)"/>

    <PropertyGroup>
      <ApplicationManifest>$(_UpdatedApplicationManifestPath)</ApplicationManifest>
    </PropertyGroup>

    <ItemGroup>
      <FileWrites Include="$(_UpdatedApplicationManifestPath)"/>
    </ItemGroup>
  </Target>
```

Reply:
I'll put this target to Arcade SDK, but for now you can put it in Roslyn's Imports.targets.